APPIUM.md

```diff
@@ -58,7 +71,17 @@ Command sample to run tests on the "iPhone 6" simulator with iOS 8.4:
  > node cordova-medic/medic/medic.js appium --app test-app --platform ios --deviceName "iPhone 6" --platformVersion 8.4 --plugins "cordova-plugin-camera cordova-plugin-contacts"
 ```
 
+When running on real iOS device, please make sure that it is located in the same network with the machine you're launching tests from and that the `UI Automation` option is enabled in `Settings -> Developer` menu.
+Please note the `--device` argument which tells Appium that we want to run the tests on a real device. You also need to specify a device UDID here.
+
+Command sample to run tests on the real iPad 2 with iOS 8.1:
+
+``` shell
+ > node cordova-medic/medic/medic.js appium --app test-app --platform ios --deviceName "iPad 2" --platformVersion 8.1 --plugins "cordova-plugin-camera cordova-plugin-contacts" --device --udid <device UDID>
+```
+
 [appium_docs]:     https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/ios-webkit-debug-proxy.md
 [brew]:            http://brew.sh/
 [plugin_camera]:   https://github.com/apache/cordova-plugin-camera
 [plugin_contacts]: https://github.com/apache/cordova-plugin-contacts
+[appium_docs]:     http://appium.io/getting-started.html
```

Comment:
There is already [appium_docs] defined so consider renaming it. Also you may directly point it to 'http://appium.io/getting-started.html#requirements'.